server/server.js

```diff
@@ -0,0 +1,9 @@
+var express = require("express");
```

Comment:
Do we need a server file here yet? It seems like this pull request is too large, you probably definitely need to rebase.